torch/onnx/_internal/exporter/_compat.py

```diff
@@ -50,7 +50,7 @@ def export_compat(
     verbose: bool | None = None,
     input_names: Sequence[str] | None = None,
     output_names: Sequence[str] | None = None,
-    opset_version: int | None = None,
+    opset_version: int | None = 18,
```

Comment:
Maybe set it to https://github.com/pytorch/pytorch/blob/aafc4b6188b70cf808f756f23b1a05355bcb7696/torch/onnx/_internal/exporter/_constants.py#L4